doc/sphinx/source/quickstart/installation.rst

```diff
@@ -363,7 +399,7 @@ Software installation
 
 Once all prerequisites are fulfilled, ESMValTool can be installed by running
 the following commands in the directory containing the ESMValTool source code
-(called ESMValTool if you did not choose a different name):
+(called ``ESMValTool`` if you did not choose a different name):
 
 .. code-block:: bash
 
```

Comment:
Perhaps add a line on what the "-e" is for. Or, perhaps use "--editable" for a bit more of a hint.

Reply:
Changed the option and added extra information